components/atom/tag/src/Actionable/Container.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Component treated as an anchor when href is defined
+ */
+const ActionableTagContainer = ({Link, href, target, children, ...props}) => {
+  return href
+    ? <Link href={href} target={target} {...props}>{children}</Link>
```

Comment:
Whats the goal of pass a Link component instead of render a link here I don't understand this part really well :/

Reply:
Maybe we could use always <a> tag here and dismiss pass a component as a prop that looks a bit strange :s

Reply:
You might want to use another kind of link-alike component, for example, a react-router link